flask_project/campaign_manager/views.py

```diff
@@ -495,21 +495,29 @@ def get_campaign_function(functions):
     function_dict = {}
```

Comment:
A docstring would be great. I agree you aren't the original author but you did improve/refactor/edit this code. 
What does this function do? From the name, it gets function (singular) of a campaign and a parameter `functions` (plural) is passed as argument. What is `functions`? An object? It's a database result? (l.572)
I think the name is misleading as it doesn't get the campaign functions. It prepares a dict (an object) for the view to use it. 
Please tell me if I'm wrong.

Reply:
I have shifted these functions to campaign_serializers.py and added a docstring to make the function more elaborate.